internal/app/wwctl/container/exec/child/main.go

```diff
@@ -21,9 +21,6 @@ import (
 )
 
 const exitEval = `$(VALU="$?" ; if [ $VALU == 0 ]; then echo write; else echo discard; fi)`
-const msgStr = `Container image is rebuilt depending on the exit status of the last command.
-
```

Comment:
This message is still needed as when the shell fails (other exit as 0) the container isn't rebuilt.

Reply:
I had removed it due to #1489; but, if you'd still like some messaging around it, I've moved it to `shell`, and also adjusted the later messaging at build-time a little bit.